showcases/angular-showcase/package.json

```
@@ -19,7 +19,7 @@
     "@angular/platform-browser": "~13.3.12",
     "@angular/platform-browser-dynamic": "~13.3.12",
     "@angular/router": "~13.3.12",
-    "@db-ui/foundations": "*",
+    "@db-ui/foundations": "file:../../packagesi/foundations",
```

@nmerget Is there a typo in the path ("packages**i**")?
```suggestion
    "@db-ui/foundations": "file:../../packages/foundations",
```

---

fixed